Gems/ROS2/Code/Source/RobotImporter/URDF/VisualsMaker.cpp

```diff
@@ -26,8 +26,7 @@
 namespace ROS2
```

Comment:
are LmbrCentral headers still required? Also, check CMake Gem target dependencies, if the Legacy one is still needed.

Reply:
Thank you for pointing that out. The unused headers are removed. Gem target dependencies cannot be changed due to colliders which are still using it.